pkg/serverless/trace/inferredspan/span_enrichment_test.go

```diff
@@ -639,17 +639,39 @@ func TestEnrichInferredSpanWithEventBridgeEvent(t *testing.T) {
 	span := inferredSpan.Span
 	assert.Equal(t, uint64(7353030974370088224), span.TraceID)
 	assert.Equal(t, uint64(8048964810003407541), span.SpanID)
-	assert.Equal(t, formatISOStartTime("2017-12-22T18:43:48Z"), span.Start)
+	assert.Equal(t, int64(100*1e6), span.Start)
 	assert.Equal(t, "eventbridge", span.Service)
 	assert.Equal(t, "aws.eventbridge", span.Name)
-	assert.Equal(t, "eventbridge.custom.event.sender", span.Resource)
+	assert.Equal(t, "testBus", span.Resource)
 	assert.Equal(t, "web", span.Type)
 	assert.Equal(t, "aws.eventbridge", span.Meta[operationName])
 	assert.Equal(t, "eventbridge.custom.event.sender", span.Meta[resourceNames])
 	assert.Equal(t, "testdetail", span.Meta[detailType])
 	assert.True(t, inferredSpan.IsAsync)
 }
 
+func TestEnrichInferredSpanWithEventBridgeEventNoBus(t *testing.T) {
+	var eventBridgeEvent events.EventBridgeEvent
+	_ = json.Unmarshal(getEventFromFile("eventbridge-no-bus.json"), &eventBridgeEvent)
+	inferredSpan := mockInferredSpan()
+	inferredSpan.EnrichInferredSpanWithEventBridgeEvent(eventBridgeEvent)
+	span := inferredSpan.Span
+	assert.Equal(t, uint64(7353030974370088224), span.TraceID)
+	assert.Equal(t, uint64(8048964810003407541), span.SpanID)
```

Comment:
I don't think we neeed to test trace and span ids here, we just overwrite later anyway.